client/src/components/CareerAndInterestsModal.js

```diff
@@ -0,0 +1,207 @@
+import React, { Component } from 'react';
+import {Button, Modal, Divider, Step, Segment } from 'semantic-ui-react';
+import PooledSingleSelectDropdown from "./PooledSingleSelectDropdown"
+import PooledMultiSelectDropdown from './PooledMultiSelectDropdown';
+import { makeCall } from "../apis";
+
+/*
+    props:
+    - modalOpen
+    - closeModal
+    - getInput
+*/
+export default class CareerAndInterestsModal extends Component {
+    constructor(props){
+        super(props)
+        // props are currently not being used to populate input fields
+        // this can be added as a future enhancement
+        this.state = {
+            // Job Title
+            existingJobTitleId: '',
+            existingJobTitleName: '',
+            newJobTitle: '',
+            // Company
+            existingCompanyId: '',
+            existingCompanyName: '',
+            newCompany: '',
+            // Interests
+            existingInterests: [],
+            newInterests: []
+        }
+        this.getJobTitleInput = this.getJobTitleInput.bind(this)
+        this.getCompanyInput = this.getCompanyInput.bind(this)
+        this.getInterestsInput = this.getInterestsInput.bind(this)
+        this.submit = this.submit.bind(this)
+        this.submitReady = this.submitReady.bind(this)
+    }
+
+    async componentWillMount() {
+        let result = await makeCall(null, '/drop/countries', 'get')
+        this.setState({countryOptions: result.options})
+    }
+
+    getCompanyInput(selection, isNew) {
```

Comment:
When we use a `PooledSingleSelectDropdown` or `PooledMultiSelectDropdown`,, a handler such as this is necessary to lift selections/ new entries to the parent component